.env.example

```diff
@@ -1,6 +1,25 @@
+APP_NAME=Barefoot nomad
+NODE_ENV=# development | production | test
+PORT=
+SECRET=
+

```

Comment:
@karamuka Would it be good if you put these codes below the ones for development?